src/attdet.c

```diff
@@ -27,14 +27,14 @@ bool lc3_attdet_run(enum lc3_dt dt, enum lc3_srate sr,
 {
     /* --- Check enabling --- */
 
-    const int nbytes_ranges[LC3_NUM_DT][LC3_NUM_SRATE - LC3_SRATE_32K][2] = {
-            [LC3_DT_7M5] = { { 61,     149 }, {  75,     149 } },
-            [LC3_DT_10M] = { { 81, INT_MAX }, { 100, INT_MAX } },
+    const int nbytes_ranges[LC3_NUM_DT - LC3_DT_7M5][LC3_NUM_SRATE - LC3_SRATE_32K][2] = {
```

Comment:
Can you index this table by the actual value of lc3_dt ? I don't think we care the two unused entries for 2M5 and 5M, and this simplifies indexing the table in the code.

Reply:
Done in 788257d